### PR TITLE
Add github templates for bug reports and feature requests

### PR DESCRIPTION
**Pull request by DarkSmile92**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Before creating a bug report, try disabling browser extensions to see if the bug is still present. -->
+
+<!-- If you're having trouble updating your profile, it is likely because you logged in separately with GitHub & Twitter. Please check if this is the case before creating a bug report, and email yo@dev.to so we can merge your accounts. -->
```
Comment (jxom) [inline .github/ISSUE_TEMPLATE/bug_report.md:8]: Maybe remove this line?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+**Desktop (please complete the following information):**
+
+- OS, version:
+- Browser, version:
+
+**Smartphone (please complete the following information):**
+
+- Device:
+- OS, version:
+- Browser, version:
```
Comment (jxom) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]: I think we can combine these two sections, and keep the contents of the smartphone section. We could even ask the user to copy + paste their browser info from https://www.whatsmybrowser.org/.

```
**Device information:**

- [What's My Browser URL](https://www.whatsmybrowser.org/):

OR

- Device:
- OS & Version:
- Browser, version:
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (DarkSmile92) [reply .github/ISSUE_TEMPLATE/bug_report.md:8]: c&p... :D removed! (but I'm allowed to do so, no worries)

**Comment by jxom** [comment]
Love it!!!

**Pull request merged**